scripts/file_utils.py

```diff
@@ -13,6 +13,11 @@
 import os
 # CCPP framework imports
 from parse_tools import CCPPError, ParseInternalError
+#XXgoldyXX: v Crap required to support python 2
```

Comment:
Suggest to remove Python 2 support, definitely not needed from our side (UFS, SCM).

Reply:
Over in CIME, I think we decided to change the shebang from `python` to `python3` for our executable scripts.
Now that I think about it, I don't know why files such as this one should even have a shebang -- it isn't even executable.
Thoughts on changing the shebang for executable scripts?